frontend/packages/dev-console/src/components/topology/TopologyControlBar.tsx

```diff
@@ -16,7 +16,14 @@ interface TopologyControlBarProps {
 }
 
 const TopologyControlBar: React.FC<TopologyControlBarProps> = ({ visualization }) => {
-  const layout = visualization.getGraph()?.getLayout() ?? 'COLA_LAYOUT';
+  const [layout, setLayout] = React.useState<string>(
+    visualization?.getGraph()?.getLayout() ?? 'COLA_LAYOUT',
```

Comment:
I expect you want to use the const here:

```suggestion
    visualization?.getGraph()?.getLayout() ?? COLA_LAYOUT,
```

Reply:
Oh see now it was previously also this string.. But I expect we should fix this anyway :smirk: